### ISSUE TITLE
Rocky 9 quickstart documentation does not work

### ISSUE THREAD

**Issue by gustavoberman** [warewulf/warewulf#1179]
### Steps to reproduce

On a new rocky9 minimal install and following the instructions at https://warewulf.org/docs/main/quickstart/el.html I have this result when trying to boot the node:

![Screenshot_rocky9-2_2024-04-06_15:44:47](https://github.com/warewulf/warewulf/assets/10926437/e9b85e88-72fc-45e7-b55b-06e350f5ca7b)

I think some steps are missing in the documentation?

### Information on your system

wwctl version
wwctl version:	 4.5.0-1
rpc version: apiPrefix:"rc1" apiVersion:"1" warewulfVersion:"4.5.0-1"

cat /etc/os-release 
NAME="Rocky Linux"
VERSION="9.3 (Blue Onyx)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="9.3"
PLATFORM_ID="platform:el9"
PRETTY_NAME="Rocky Linux 9.3 (Blue Onyx)"
ANSI_COLOR="0;32"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:rocky:rocky:9::baseos"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
SUPPORT_END="2032-05-31"
ROCKY_SUPPORT_PRODUCT="Rocky-Linux-9"
ROCKY_SUPPORT_PRODUCT_VERSION="9.3"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="9.3"

cat /etc/warewulf/warewulf.conf 
WW_INTERNAL: 45
ipaddr: 172.16.0.1
netmask: 255.255.0.0
network: 172.16.0.0
warewulf:
  port: 9873
  secure: false
  update interval: 60
  autobuild overlays: true
  host overlay: true
  syslog: false
  datastore: /usr/share
  grubboot: false
dhcp:
  enabled: true
  template: default
  range start: 172.16.1.1
  range end: 172.16.1.255
  systemd name: dhcpd
tftp:
  enabled: true
  tftproot: /var/lib/tftpboot
  systemd name: tftp
  ipxe:
    "00:00": undionly.kpxe
    "00:07": ipxe-snponly-x86_64.efi
    "00:09": ipxe-snponly-x86_64.efi
    00:0B: arm64-efi/snponly.efi
nfs:
  enabled: true
  export paths:
  - path: /home
    export options: rw,sync
    mount options: defaults
    mount: true
  - path: /opt
    export options: ro,sync,no_root_squash
    mount options: defaults
    mount: false
  systemd name: nfs-server
container mounts:
- source: /etc/resolv.conf
  dest: /etc/resolv.conf
  readonly: true
paths:
  bindir: /usr/bin
  sysconfdir: /etc
  localstatedir: /var/lib
  ipxesource: /usr/share/ipxe
  srvdir: /var/lib
  firewallddir: /usr/lib/firewalld/services
  systemddir: /usr/lib/systemd/system
  wwoverlaydir: /var/lib/warewulf/overlays
  wwchrootdir: /var/lib/warewulf/chroots
  wwprovisiondir: /var/lib/warewulf/provision
  wwclientdir: /warewulf

wwctl container list
  CONTAINER NAME  NODES  KERNEL VERSION                CREATION TIME        MODIFICATION TIME    SIZE       
  rockylinux-9    1      5.14.0-362.13.1.el9_3.x86_64  06 Apr 24 15:37 -03  31 Dec 69 21:00 -03  778.3 MiB  

### General information

- [X] I have run `wwctl version` and reported the contents of `/etc/os-release`
- [X] I have searched the issues of this repo and believe this is not a duplicate
- [X] I have captured and reported relevant error messages and logs

**Comment by anderbubble** [comment]
I'll take a look. Thanks, @gustavoberman!

**Comment by garthand** [comment]
I did a quick demo run of the quickstart documentation and found two things that might help:

1. There's a lot of fighting with iPXE that might happen for a variety of reasons; (secure boot, wrong iPXE files being provided by default, iPXE builds with missing features, etc). You can sidestep this all by doing a grubboot=true (which is documented elsewhere but not in the quickstart)
2. The quickstart documentation never instructs the user to run wwctl overlay build and so the VFS will fail to mount as /var/lib/warewulf/provision/overlays/$node/__RUNTIME__.img will not be found

**Comment by anderbubble** [comment]
Thanks, @garthand!

**Comment by anderbubble** [comment]
> You can sidestep this all by doing a grubboot=true

It's still pretty early days for grubboot in Warewulf. I expect we'll switch to it by default in the not too distant future (probably v4.7.0) and the quickstart will use it then.

> The quickstart documentation never instructs the user to run wwctl overlay build

This is _supposed_ to be automatic; but it was broken. We fixed overlay autobuild in v4.5.6; but, just to clear it up for people who might still be on older versions, I'll add an explicit mention of it here.

> Running wwctl configure --all doesn't configure the ssh keys

This was fixed in v4.5.3, and was only broken in v4.5.2; so I think I'll leave this particular issue out of the quickstart.
